src/v/pandaproxy/schema_registry/json.cc

```diff
@@ -136,6 +136,24 @@ json::Pointer to_json_pointer(std::string_view sv) {
     return candidate;
 }
 
+// helper to convert a jsoncons::ojson to a rapidjson::Document
+json::Document to_json_document(const jsoncons::ojson& oj) {
```

Comment:
nit: I'd name this something like `to_rapidjson` or `to_rpjson_document` to signal in the name that this is converting between different library representations